src/index.tsx

```diff
@@ -1,10 +1,13 @@
 import React, { useState, useEffect, useRef, useCallback } from 'react'
 import { largestRect } from 'rect-scaler'
 
+type FunctionBoxClassNameType = (child: React.ReactNode) => string;
```

Comment:
Could we avoid passing around and introspecting React nodes like this? The pattern is [discouraged / deprecated](https://twitter.com/sebmarkbage/status/1501973552362369031)

Maybe instead, we only pass the index of the child?